ethcore/src/json_tests/difficulty.rs

```diff
@@ -37,7 +37,6 @@ pub fn json_difficulty_test<H: FnMut(&str, HookType)>(
 	for (name, test) in tests.into_iter() {
 		start_stop_hook(&name, HookType::OnStart);
 
-		flush!("   - {}...", name);
 		println!("   - {}...", name);
```

Comment:
> println!("   - {}...", name);

I'm not sure why the line is printed. 🤔 It seems okay to delete the line as the `info!` prints the same information.

Reply:
take a look at the previous attempt here #11208